guides/keras_cv/retina_net_overview.py

```diff
@@ -0,0 +1,281 @@
+)
+
+augmentation_layers = [
+    # keras_cv.layers.RandomShear(x_factor=0.1, bounding_box_format='xywh'),
+    # TODO(lukewood): add color jitter and others
+]
+
+
+def augment(sample):
+    for layer in augmentation_layers:
+        sample = layer(sample)
+    return sample
+
+
+train_ds = train_ds.map(augment, num_parallel_calls=tf.data.AUTOTUNE)
+visualize_dataset(train_ds, bounding_box_format="xywh")
+
+"""
+Great!  We now have a bounding box friendly augmentation pipeline.
+
+Next, let's unpackage our inputs from the preprocessing dictionary, and prepare to feed
+the inputs into our model.
+"""
+
+
+def dict_to_tuple(inputs):
+    return inputs["images"], inputs["bounding_boxes"]
+
+
+train_ds = train_ds.map(unpackage_dict, num_parallel_calls=tf.data.AUTOTUNE)
+val_ds = val_ds.map(unpackage_dict, num_parallel_calls=tf.data.AUTOTUNE)
+
+train_ds = train_ds.prefetch(tf.data.AUTOTUNE)
+val_ds = val_ds.prefetch(tf.data.AUTOTUNE)
+
+"""
+Our data pipeline is now complete.  We can now move on to model creation and training.
+"""
+
+"""
+## Model creation
+
+We'll use the KerasCV API to construct a RetinaNet model.  In this tutorial we use
+a pretrained ResNet50 backbone using weights.  In order to perform fine-tuning, we
+freeze the backbone before training.  When `include_rescaling=True` is set, inputs to
+the model are expected to be in the range `[0, 255]`.
+"""
+
+model = keras_cv.models.RetinaNet(
+    # number of classes to be used in box classification
+    classes=20,
+    # For more info on supported bounding box formats, visit
+    # https://keras.io/api/keras_cv/bounding_box/
+    bounding_box_format="xywh",
+    # KerasCV offers a set of pre-configured backbones
+    backbone="resnet50",
+    # Each backbone comes with multiple pre-trained weights
+    # These weights match the weights available in the `keras_cv.model` class.
+    backbone_weights="imagenet",
+    # include_rescaling tells the model whether your input images are in the default
+    # pixel range (0, 255) or if you have already rescaled your inputs to the range
+    # (0, 1).  In our case, we feed our model images with inputs in the range (0, 255).
+    include_rescaling=True,
+)
+
+"""
+That is all it takes to construct a KerasCV RetinaNet.  The RetinaNet accepts tuples of
+dense image Tensors and ragged bounding box Tensors to `fit()` and `train_on_batch()`
+This matches what we have constructed in our input pipeline above.
+
+The RetinaNet `call()` method outputs two values: training targets and inference targets.
+In this guide, we are primarily concerned with the inference targets.  Internally, the
+training targets are used by `keras_cv.losses.ObjectDetectionLoss()` to train the
+network.
+"""
+
+"""
+## Optimizer
+
+For training, we use a SGD optimizer with a piece-wise learning rate schedule
+consisting of a warm up followed by a ramp up, then a ramp.
+Below, we construct this using a `keras.optimizers.schedules.PiecewiseConstantDecay`
```

Comment:
I had missed this -- the text does not match the code. The code does not currently use a schedule.